docs/faq.md

```diff
@@ -178,4 +179,9 @@ create datamap ag1 on table gdp21 using 'preaggregate' as select cntry, sum(gdp)
 select cntry,sum(gdp) from gdp21,pop1 where cntry=ctry group by cntry;
 ```
 
+## Why all executors are returning success even after some of the query failed?
```

Comment:
language is not correct. its not a query, it a command.We need to say why executor does not retry and why the signal sent to driver is not interpreted as failed